lib/hooks/useOrderLine/useOrderLine.js

```diff
@@ -0,0 +1,34 @@
+import { useQuery } from 'react-query';
+
+import {
+  useNamespace,
+  useOkapiKy,
+} from '@folio/stripes/core';
+
+import { LINES_API } from '../../constants';
+
+export const useOrderLine = (lineId, options = {}) => {
+  const ky = useOkapiKy();
+  const [namespace] = useNamespace({ key: 'purchase-order-line' });
+
+  const { enabled = true, ...queryOptions } = options;
+
+  const {
+    data,
+    isFetching,
+    isLoading,
+  } = useQuery(
+    [namespace, lineId],
+    async () => ky.get(`${LINES_API}/${lineId}`).json(),
```

Comment:
it doesn't need to be async here, you don't await anything.
and it's just a suggestion - passing params as object looks more readable to me, for example:

```javascript
useQuery(
    {
      queryKey: [namespace, lineId],
      queryFn: () => ky.get(`${LINES_API}/${lineId}`).json(),
      enabled: enabled && Boolean(lineId),
      ...queryOptions,
    },
  )
```

Reply:
Thank you @mariia-aloshyna! I've removed the `async` declaration.
As for the second point maybe you're right, but it would be more consistent with other hooks, that are already implemented if to left as is. Besides, IDE solves the problem of readability :)

![image](https://github.com/folio-org/stripes-acq-components/assets/88109087/f4462710-a923-4e8f-ab7a-0ec6e2dd691c)